elyra/pipeline/processor_kfp.py

```diff
@@ -67,8 +97,22 @@ def process(self, pipeline):
             self.log.info("Kubeflow Pipeline successfully compiled.")
             self.log.debug("Kubeflow Pipeline was created in %s", pipeline_path)
 
-            # Upload the compiled pipeline and create an experiment and run
-            client = kfp.Client(host=api_endpoint)
+            # Upload the compiled pipeline, create an experiment and run
+
+            if (api_endpoint_username is not None) and (api_endpoint_password is not None):
+                endpoint = api_endpoint.replace('/pipeline', '')
+                session_cookie = self.get_auth_session_cookie(endpoint, api_endpoint_username, api_endpoint_password)
+
+                # TODO: handle case no session cookie found --> authentication error?
+                # When session cookie is none, we get a making request error, reason: No host specified
```

Comment:
what happens if you return an empty string instead of none?

Reply:
same behavior